gamatrix-gog.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+# TODO: copy max players to other games with the same sanitized title
```

Comment:
Don't bother leaving this, just make an issue!

Reply:
Yeah, was being lazy 😞